backend/packages/Upgrade/src/api/repositories/MetricRepository.ts

```diff
@@ -32,7 +33,9 @@ export class MetricRepository extends Repository<Metric> {
   public async findMetricsWithQueries(ids: string[]): Promise<Metric[]> {
```

Comment:
Changes over here are unnecessary. Because the query does more than what the function name suggests.